<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cal Cap Lending Commision Calculator</string>
  </property>
  <property name="windowIcon">
   <iconset theme="hello">
    <normaloff>../../PythonApplication7/PythonApplication7</normaloff>../../PythonApplication7/PythonApplication7</iconset>
  </property>
  <widget class="QLabel" name="term_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Loan Term Length:</string>
   </property>
  </widget>
  <widget class="QLabel" name="fico_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>115</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Client's FICO Score:</string>
   </property>
  </widget>
  <widget class="QLabel" name="ltv_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Property Value:</string>
   </property>
  </widget>
  <widget class="QLabel" name="entity_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>185</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Entity or Non:</string>
   </property>
  </widget>
  <widget class="QLabel" name="amount_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Loan Amount:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="term_input">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>6 Months</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12 Months</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>18 Months</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>24 Months</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="fico_input">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>115</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>620-659</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>660-699</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>700+</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="amount_input">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>$</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>60</y>
     <width>561</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="title_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>26</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Commission Calculator</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ltv_input">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>$</string>
   </property>
  </widget>
  <widget class="QPushButton" name="go_button">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>GO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reset_button">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>360</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>RESET</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="output_box">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>101</y>
     <width>281</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="entity_input">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>185</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Entity</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Non-Entity</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>255</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Cash-Out:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Foreign National:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>325</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Condo:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cash_out">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>255</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>No</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Yes</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="fnational">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>290</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>No</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Yes</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="condo">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>325</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>No</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Yes</string>
    </property>
   </item>
  </widget>
 </widget>
 <tabstops>
  <tabstop>term_input</tabstop>
  <tabstop>fico_input</tabstop>
  <tabstop>ltv_input</tabstop>
  <tabstop>entity_input</tabstop>
  <tabstop>amount_input</tabstop>
  <tabstop>cash_out</tabstop>
  <tabstop>fnational</tabstop>
  <tabstop>condo</tabstop>
  <tabstop>go_button</tabstop>
  <tabstop>reset_button</tabstop>
  <tabstop>output_box</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
